404.html

```diff
@@ -42,7 +42,41 @@ <h2>Move along.</h2>
 </div>
 
 <script>
-$(function () {
-    // TODO add url to the report button
-  };)
+  $(function () {
+    function updateGitHref() {
+      let githubLink = "https://github.com/ros-infrastructure/rosindex/issues/new?labels=bug&template=form404.yml";
+
+      // Get current url
+      var currentUrl = window.location.href;
+      if (!currentUrl) {
+        currentUrl = "URL+not+available";
+      }
+
+      // Get the url that brought us here
+      var referrerUrl = document.referrer;
+      if (!referrerUrl) {
+        referrerUrl = "Referrer+URL+not+available";
```

Comment:
Can you make this clearly ask the user to fill it in. More like `<FAILED TO AUTOMATICALLY DETECT REFERRER PLEASE FILL>` such that it will be very clear to fill it in. With HTTPS most places this often will be empty. 

And do likewise with the url detection?

Reply:
Hi @tfoote 

The Referrer URL input and 404 page input are already marked as required, with only the description remaining optional.

[GitHub Docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms)

I've updated the form so that if the Referrer URL or 404 page are not detected, their fields will remain blank, preventing the form from being submitted.

I've added the placeholder text "FAILED TO AUTOMATICALLY DETECT REFERRER PLEASE FILL" and "FAILED TO AUTOMATICALLY DETECT 404 PAGE URL PLEASE FILL" to the input fields. Placing it as the placeholder ensures that users are prompted to fill in the field if it's left blank. If it were added directly to the input value, the form might allow submission even if the field is technically empty.

Should i add  "FAILED TO AUTOMATICALLY DETECT REFERRER PLEASE FILL" and "FAILED TO AUTOMATICALLY DETECT 404 PAGE URL PLEASE FILL" as the text input value instead of being the placeholders ?

Reply:
Hi @tfoote

The Referrer URL input and 404 page input are already marked as required, with only the description remaining optional.

[GitHub Docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms)

I've updated the form so that if the Referrer URL or 404 page are not detected, their fields will remain blank, preventing the form from being submitted.

I've added the placeholder text "FAILED TO AUTOMATICALLY DETECT REFERRER PLEASE FILL" and "FAILED TO AUTOMATICALLY DETECT 404 PAGE URL PLEASE FILL" to the input fields. Placing it as the placeholder ensures that users are prompted to fill in the field if it's left blank. If it were added directly to the input value, the form might allow submission even if the field is technically empty.

Should i add "FAILED TO AUTOMATICALLY DETECT REFERRER PLEASE FILL" and "FAILED TO AUTOMATICALLY DETECT 404 PAGE URL PLEASE FILL" as the text input value instead of being the placeholders ?